images/win/scripts/ImageHelpers/InstallHelpers.ps1

```diff
@@ -49,25 +49,29 @@ function Install-Binary
         $filePath = "msiexec.exe"
     }
 
+    $installStartTime = Get-Date
     try
     {
         Write-Host "Starting Install $Name..."
         $process = Start-Process -FilePath $filePath -ArgumentList $ArgumentList -Wait -PassThru
-
         $exitCode = $process.ExitCode
+        $installCompleteTime = [math]::Round(($(Get-Date) - $installStartTime).TotalSeconds, 2)
```

Comment:
nitpick: Why just don't cast to [int]?

Reply:
[int]/[int16]/[int32] so as *.ToInt/*.ToInt16/*.ToInt32 not rounding but taking just whole integer value so might output just 0.
ToDouble decimal .Net method way more complicated.
I wanted to round measured time to second period of milliseconds for better logging and visibility.

Reply:
`[int]/[int16]/[int32] so as .ToInt/.ToInt16/*.ToInt32 not rounding but taking just whole integer value so might output just 0.`
Not exactly, it's called banker's rounding.
```
PS > [int]0.6
1
PS > [int]0.2
0
```

`ToDouble decimal .Net method way more complicated. `
The current type is already Double.

Reply:
Anyway. Will it be better to round and output like this?
For example for cases like following:
- https://github.visualstudio.com/virtual-environments/_build/results?buildId=116599&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=5431112d-2b61-5a5f-7042-ef698f761043&l=5550
- https://github.visualstudio.com/virtual-environments/_build/results?buildId=116599&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=5431112d-2b61-5a5f-7042-ef698f761043&l=5555

Reply:
LGTM.